packages/myst-to-jats/tests/basic.spec.ts

```diff
@@ -75,7 +75,14 @@ beforeEach(() => {
 describe('Basic JATS body', () => {
   const cases = [...loadCases('basic.yml'), ...loadCases('siunit.yml')];
   test.each(cases.map((c): [string, TestCase] => [c.title, c]))('%s', async (_, { tree, jats }) => {
-    const pipe = unified().use(mystToJats, SourceFileKind.Article);
+    const pipe = unified().use(
+      mystToJats,
+      SourceFileKind.Article,
+      undefined,
+      undefined,
+      undefined,
+      { spaces: 'flat' },
+    );
```

Comment:
@fwkoch, have run into this elsewhere - the API on this plugin is pretty hard to use - we should improve it at some point!

Reply:
Yes - I agree... I feel like there could be lots of cleanup making APIs consistent across MyST (and also we do a lot of wrapping functions in plugins, but then sometimes use the plugins and other times use the functions themselves... would be great to have a little more intentionality about all that as well...)